_includes/layout/base/html-head.html

```diff
@@ -8,6 +8,11 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
 <title>{% capture title %}{% translate title %}{% endcapture %}{% if title != '' %}{{ title }}{% else %}{{ page.title }}{% endif %}</title>
 {% capture metadescription %}{% translate metadescription %}{% endcapture %}{% if metadescription != '' %}<meta name="description" content="{{ metadescription }}">{% endif %}
+
+{% comment %}<!-- load Font Awesome but hide it if JS is disabled -->{% endcomment %}
+<link rel="stylesheet" href="/css/font-awesome-4.4.0/css/font-awesome.min.css">
+<noscript><style>.fa { display: none !important; }</style></noscript>
```

Comment:
@saivann Agreed.  I'm a pretty serious NoScript user and I've become used to seeing those broken unicode characters on pages that use Font Awesome or something similar, so I imagine other users won't be upset to see them on Bitcoin.org.

I'll remove the lines.